JSDemos/Demos/TreeList/Resizing/description.md

```diff
@@ -1 +1,26 @@
-The TreeList allows a&nbsp;user to&nbsp;resize columns at&nbsp;runtime. They can be&nbsp;resized in&nbsp;two different modes: by&nbsp;changing the width of&nbsp;the next column or&nbsp;the width of&nbsp;the component. To&nbsp;switch the current mode, use the &laquo;Column resizing mode&raquo; select box.
\ No newline at end of file
+The TreeList sets equal widths for all columns, but you can change the widths as described below:
+
+### Specify Custom Column Widths
+You can set custom widths for all or individual columns. Individual settings override common settings. Use the following properties to specify the widths:
+
+* [columnWidth](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#columnWidth) / **columns[]**.[width](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#width)               
+Specify a width for all or individual columns.
+
+* [columnMinWidth](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#columnMinWidth) / **columns[]**.[minWidth](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#minWidth)                        
+Specify a minimum width for all or individual columns.
+
+### Auto-Adjust Column Widths to the Content
```

Comment:
Content is uncountable

```suggestion
### Auto-Adjust Column Widths to Content
```

Reply:
Uncountable nouns can be used with "the".

Reply:
They can be indeed. But why? Are you referring to specific content here? Why the need for this emphasis?